core/framework/extractors/latex_extractor.py

```
@@ -0,0 +1,494 @@
+                
+                # If not tar, check if it looks like LaTeX
+                if not is_tar:
+                    try:
+                        # Try to decode as text
+                        text_sample = initial_bytes.decode('utf-8', errors='ignore')
+                        # Check for common LaTeX patterns
+                        is_latex = ('\\documentclass' in text_sample or 
+                                  '\\begin{document}' in text_sample or
+                                  '\\section' in text_sample or
+                                  '\\usepackage' in text_sample)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)